src/main/java/org/jboss/tools/intellij/openshift/utils/ConfigHelper.java

```diff
@@ -42,40 +43,18 @@ public static Config safeLoadKubeConfig() {
 
     public static Config loadKubeConfig() throws IOException {
         File f = new File(getKubeConfigPath());
-        if (f.exists()) {
+        if (Files.exists(f.toPath())) {
             return KubeConfigUtils.parseConfig(f);
         } else {
             return new ConfigBuilder().build();
         }
     }
 
-    public static boolean isKubeConfigParsable() {
```

Comment:
What's the rationale for this removal. Is this because this method is not used ?

Reply:
yes, while on working on that class, i noticed these methods are not used, so i removed it.